pkg/utils/tekton/controller.go

```diff
@@ -197,31 +197,45 @@ func (k KubeController) RunPipeline(g PipelineRunGenerator, taskTimeout int) (*v
 			}
 		}
 	}
-
 	return k.createAndWait(pr, taskTimeout)
 }
 
-func createPVC(pvcs v1.PersistentVolumeClaimInterface, pvcName string) error {
+func CreatePVC(pvcs v1.PersistentVolumeClaimInterface, pvcName string, volumeAccessMode string, volumeSize string) error {
```

Comment:
This should be in common IMHO. Also, the declaration should be something like:
```suggestion
func (k KubeController) CreatePVC(pvcName, volumeSize string, volumeAccessMode corev1.PersistentVolumeAccessMode) error {
```

Reply:
this PR is a draft , new one will be created , 
please ignore this